sdk/netapp/arm-netapp/README.md

```diff
@@ -108,6 +107,8 @@ If you'd like to contribute to this library, please read the [contributing guide
 
 - [Microsoft Azure SDK for JavaScript](https://github.com/Azure/azure-sdk-for-js)
 
+![Impressions](https://azure-sdk-impressions.azurewebsites.net/api/impressions/azure-sdk-for-js%2Fsdk%2Fnetapp%2Farm-netapp%2FREADME.png)
```

Comment:
why we have this?

Reply:
Probably need to update codegen if it is still adding this? Impression pixels have been removed in PR #32887

Reply:
yes, here is the codegen issue to track https://github.com/Azure/autorest.typescript/issues/3075